utils/eliteDangerousAPI.js

```diff
@@ -45,18 +62,33 @@ function getEliteStatus() {
 function getGalnetFeed() {
   return new Promise(async (fulfill, reject) => {
     try {
-      let url = "https://www.alpha-orbital.com/galnet-feed";
-      fulfill(JSON.parse(await request(url)));
+      axios.get("https://www.alpha-orbital.com/galnet-feed")
+        .then(function (response) {
+          fulfill(JSON.parse(response.data));

```

Comment:
`response.data` comes pre-parsed.
```suggestion
          fulfill(response.data);

```